torch/csrc/jit/passes/quantization.cpp

```diff
@@ -30,7 +30,7 @@ using NameModuleVector = std::vector<std::pair<std::string, script::Module>>;
 // Map of quantization parameter name and value
 // for example _scale, _zero_point,
 // _scalar_type and _axis(for per channel quantization)
-using QParamMap = std::unordered_map<std::string, IValue>;
+using QParamVector = std::vector<std::pair<std::string, IValue>>;
```

Comment:
Can you point to a test that checks per channel quantization?

Reply:
https://github.com/pytorch/pytorch/blob/master/test/test_jit.py#L1291